tests/tests/cluster/test-kyma-modules.spec.js

```diff
@@ -0,0 +1,156 @@
+/// <reference types="cypress" />
+
+context('Test Kyma Modules views', () => {
+  before(() => {
+    cy.loginAndSelectCluster();
+  });
+
+  it('Test Feature card for Modules', () => {
+    cy.wait(2000);
+
+    cy.get('ui5-card')
+      .contains('Installed Modules')
+      .should('be.visible');
+
+    cy.get('ui5-card')
+      .contains('Modify Modules')
+      .click();
+
+    cy.url().should('match', /.*\/kymamodules/);
+  });
+
+  it('Test adding Modules', () => {
+    cy.get('ui5-panel')
```

Comment:
Please add checking at the beginning of this `it` if the release channel for Kyma CR is displayed, and if `No Modules` section in the panel is displayed.